lib/Core/Executor.cpp

```diff
@@ -3826,14 +3826,15 @@ void Executor::runFunctionAsMain(Function *f,
     SearchTree::save(interpreterHandler->getOutputFilename("tree.dot"));
     SearchTree::deallocate();
 
-#ifdef SUPPORT_Z3
-    // Print interpolation time statistics
-    if (InterpolationStat)
-      interpreterHandler->assignSubsumptionStats(ITree::getInterpolationStat());
-#endif
-
     delete interpTree;
     interpTree = 0;
+
+#ifdef SUPPORT_Z3
+		// Print interpolation time statistics
+		if (InterpolationStat)
```

Comment:
Please correct the indentation here. It seems the code is indented more compared to lines before the `#ifdef`.